plugins/thunar-apr/thunar-apr-image-page.c

```diff
@@ -173,7 +172,9 @@ thunar_apr_image_page_init (ThunarAprImagePage *image_page)
   /* add labels for the Exif info */
   for (n = 0; n < G_N_ELEMENTS (TAIP_EXIF); ++n)
     {
-      label = gtk_label_new (_(TAIP_EXIF[n].name));
+      const gchar *exif_title = exif_tag_get_title ( TAIP_EXIF[n] );
```

Comment:
1. Yes, but .. but I don't understand why. They claim that you should use exif_tag_get_title_in_ifd .. but to do this this way (IMHO) you need to use additional exif_tag_get_stuff made loop to iterate on all IFD by self. Personally, I prefer simpler version ;-)

2. Yes. " The title is a short, localized description of the tag."
http://libexif.sourceforge.net/internals/exif-tag_8c.html#6530318a3f4c3cb5020d9b55447f073a

They have localization for this languages:
http://libexif.cvs.sourceforge.net/viewvc/libexif/libexif/po/